### PR TITLE
6666 Portal improvement next ... the hell is in details

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

https://github.com/gravitee-io/issues/issues/6666

**Description**

Migrate tslint to eslint 
Try to bump all dependencies to last version

**Additional context**

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-hyfecjkdcv.chromatic.com)

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6666-improvement-2/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-portal-webui/src/index.html [hunk 1]
```diff
@@ -23,6 +23,11 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <script src="webcomponents/webcomponents-loader.js"></script>
+    <script src="@asciidoctor/asciidoctor.js"></script>
+    <script>
+      // Preload the asciidoctor dependencies before ui-component because asciidoctor not work with asynchronous esm import (or something like that)
```
Comment (ThibaudAV) [inline gravitee-apim-portal-webui/src/index.html:28]: 🧙🧙🧙🧙🧙

File: gravitee-apim-portal-webui/src/app/components/gv-button-create-application/gv-button-create-application.component.html [hunk 2]
```diff
@@ -15,4 +15,6 @@
     limitations under the License.
 
 -->
-<gv-button primary icon="code:plus">{{ 'gv-create-application.createApplication' | translate }}</gv-button>
+<gv-button link primary icon="code:plus" *ngIf="'gv-create-application.createApplication' | translate as createApplicationLabel">
```
Comment (ThibaudAV) [inline gravitee-apim-portal-webui/src/app/components/gv-button-create-application/gv-button-create-application.component.html:18]: 🧙🧙🧙🧙🧙

File: gravitee-apim-portal-webui/package.json [hunk 3]
```diff
@@ -3,8 +3,8 @@
   "version": "0.0.0",
   "description": "Gravitee.io Portal",
   "scripts": {
-    "build": "ng build --aot",
-    "build:prod": "ng build --configuration production",
+    "build": "NODE_OPTIONS=--max_old_space_size=8192 ng build",
```
Comment (ThibaudAV) [inline gravitee-apim-portal-webui/package.json:6]: 😞  the portal take 6 go in memory or more .... and I didn't find out why ... i've some idea but no real proof

**Comment by ThibaudAV** [comment]
Please have a look at SonarCoud bug about ` new windowNotification`. 
I don't know if I can delete the code or even how to check the non-regression  🙏 🥺

File: gravitee-apim-portal-webui/package.json (hunk 3 above)
Comment (gaetanmaisse) [reply gravitee-apim-portal-webui/package.json:6]: :aa:

**Review by gaetanmaisse — APPROVED** [review]

**Comment by gaetanmaisse** [comment]
Tested locally, looks ok

**Pull request merged**